edit/edit.go

```diff
@@ -256,7 +257,37 @@ func FindRuleByName(f *build.File, name string) *build.Rule {
 	if i != -1 {
 		return &build.Rule{Call: f.Stmt[i].(*build.CallExpr)}
 	}
+	return UseImplicitName(f, name)
+}
+
+// UseImplicitName returns the rule in the file if it meets these conditions:
+// - It is the only rule in the file.
+// - The passed package and rule names match.
+func UseImplicitName(f *build.File, rule string) *build.Rule {
+	if f.Path == "BUILD" {
```

Comment:
Maybe add a comment explaining this early-return. ("We don't allow empty names.")